internal/cri/server/restart.go

```diff
@@ -181,6 +181,10 @@ func (c *criService) recover(ctx context.Context) error {
 			if err := c.containerStore.Add(cntr); err != nil {
 				return fmt.Errorf("failed to add container %q to store: %w", container.ID(), err)
 			}
+			if exitCh != nil {
+				// Start exit monitor,this should run after container is added into containerStore.
```

Comment:
suggestion:

Start the exit monitor. This should run after that container has been added to the container store.